semantic-go/src/Language/Go/Tags.hs

```diff
@@ -97,11 +79,108 @@ gtags
   -> m ()
 gtags = Tags.traverse1_ @ToTags (const (pure ())) tags . Tags.Generics
 
-instance (Generic1 t, Tags.GTraversable1 ToTags (Rep1 t)) => ToTagsBy 'Generic t where
-  tags' = gtags
-
 yieldTag :: (Has (Reader Source) sig m, Has (Writer Tags.Tags) sig m) => Text -> Kind -> Loc -> Range -> m ()
 yieldTag name kind loc range = do
   src <- ask @Source
   let sliced = slice src range
   Tags.yield (Tag name kind loc (Tags.firstLine sliced) Nothing)
+
+
+instance ToTags Go.ArgumentList
+instance ToTags Go.ArrayType
+instance ToTags Go.AssignmentStatement
+instance ToTags Go.BinaryExpression
+instance ToTags Go.BlankIdentifier
+instance ToTags Go.Block
+instance ToTags Go.BreakStatement
+-- instance ToTags Go.CallExpression
+instance ToTags Go.ChannelType
+instance ToTags Go.CommunicationCase
+instance ToTags Go.CompositeLiteral
+instance ToTags Go.ConstDeclaration
+instance ToTags Go.ConstSpec
+instance ToTags Go.ContinueStatement
+instance ToTags Go.DecStatement
+instance ToTags Go.DefaultCase
+instance ToTags Go.DeferStatement
+instance ToTags Go.Dot
+instance ToTags Go.Element
+instance ToTags Go.EmptyStatement
+instance ToTags Go.EscapeSequence
+instance ToTags Go.Expression
+instance ToTags Go.ExpressionCase
+instance ToTags Go.ExpressionList
+instance ToTags Go.ExpressionSwitchStatement
+instance ToTags Go.FallthroughStatement
+instance ToTags Go.False
+instance ToTags Go.FieldDeclaration
+instance ToTags Go.FieldDeclarationList
+instance ToTags Go.FieldIdentifier
+instance ToTags Go.FloatLiteral
+instance ToTags Go.ForClause
+instance ToTags Go.ForStatement
+instance ToTags Go.FuncLiteral
+-- instance ToTags Go.FunctionDeclaration
```

Comment:
I’ve left commented-out lines for the customized instances in the alphabetized lists. It’s not super important but it provides a nice overview.